### ISSUE TITLE
Make vhs be quiet

### ISSUE THREAD

**Issue by rm-dr** [charmbracelet/vhs#189]
Exactly what it says on the tin.

Any self-respecting cli tool has a flag akin to `--quiet` that suppresses non-critical output. \
Since VHS is likely to be used in build scripts, it needs this feature more than most.

**Comment by maaslalani** [comment]
Agreed we should probably have a `--quiet` flag (maybe also a `--verbose`/`--debug`) flag.

For the time being, you can redirect all the output to `/dev/null` which should do what you want.

```bash
vhs demo.tape >/dev/null 2>&1
```

**Comment by maaslalani** [comment]
Fixed by #222 / #270
